tasks/pkg-redhat.yml

```diff
@@ -6,14 +6,21 @@
     sha256sum: 694a2ffecff85326cc08e5f1a619937999a5913171e42f166e13ec802c812085
 
 - name: Import new RPM key
-  rpm_key: key=/tmp/DATADOG_RPM_KEY_E09422B3.public state=present
+  rpm_key:
+    key: /tmp/DATADOG_RPM_KEY_E09422B3.public
+    state: present
 
 - name: Copy repo file into place
-  template: src=datadog.repo.j2 dest=/etc/yum.repos.d/datadog.repo owner=root group=root mode=0644
+  template:
+    src: datadog.repo.j2
+    dest: /etc/yum.repos.d/datadog.repo
+    owner: root
+    group: root
+    mode: 0644
 
 - name: Install pinned datadog-agent package
   yum:
-    name: datadog-agent-{{ datadog_agent_version }}
+    name: "datadog-agent-{{ datadog_agent_version }}"
```

Comment:
I don't understand why this needs to be quoted?

Reply:
As I understand it it's been the Ansible convention for a while that strings that include variables (i.e. `{{ var }}`) should be quoted.

Reply:
This is true.

Reply:
It's actually not required to be quoted here, but the convention seems to be to quote anything containing Jinja2 templating.

That being said, if the YAML value **starts** with a template `{{` it **must** be quoted or YAML will interpret it is starting a dictionary structure.

Reply:
I think we should bikeshed this a bit more before merging, because meanwhile the branch has gone stale with only *two* merge conflicts /sarcasm. ;-P